<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientW</class>
 <widget class="QWidget" name="ClientW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Приложение Сервер</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры соединения</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="address_le">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Порт:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port_le">
          <property name="text">
           <string>12224</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="connect_pb">
          <property name="text">
           <string>Соединиться</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnect_pb">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Разъединиться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="main_textedit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="message_le"/>
     </item>
     <item>
      <widget class="QPushButton" name="send_pb">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
